flask_celeryext/app.py

```diff
@@ -26,7 +26,11 @@ def setup_task_logger(logger=None, **kwargs):
 def create_celery_app(flask_app):
     """Create a Celery application."""
     celery = current_celery_app
-    celery.conf.update(flask_app.config)
+    if celery_version < '4.0':
```

Comment:
What about using `from distutils.version import StrictVersion` instead of string comparison?